### ISSUE TITLE
Model Accuracy

### ISSUE THREAD

**Issue by iqnewgnod** [weiaicunzai/pytorch-cifar100#23]
i use your model  "seresnet18"    to train cifar-100 datasets, and i try your train Details, "200epoch , init lr = 0.1 divide by 5 at 60th, 120th, 160th epochs, train for 200 epochs with batchsize 128 and weight decay 5e-4,  " but i cant get the 23.56 accuracty  rate ,  i only get 33.01 accuracy rate! 
can you help me ?

**Comment by xyy1991** [comment]
I have  the same problem

**Comment by JiyueWang** [comment]
I train seresnet32 and got 24.5 error rate, far behind

**Comment by THHHomas** [comment]
The same problem. resnet18, the top1 error is 0.3430.

**Comment by ErezBeyond** [comment]
I am getting 0.2565 top error on resnet18 with all the defaults. So 1 point off

> The same problem. resnet18, the top1 error is 0.3430.

Same here.

**Comment by 18251899066** [comment]
> i use your model "seresnet18" to train cifar-100 datasets, and i try your train Details, "200epoch , init lr = 0.1 divide by 5 at 60th, 120th, 160th epochs, train for 200 epochs with batchsize 128 and weight decay 5e-4, " but i cant get the 23.56 accuracty rate , i only get 33.01 accuracy rate!
> can you help me ?

i got same problem

**Comment by Pengfight** [comment]
> i use your model "seresnet18" to train cifar-100 datasets, and i try your train Details, "200epoch , init lr = 0.1 divide by 5 at 60th, 120th, 160th epochs, train for 200 epochs with batchsize 128 and weight decay 5e-4, " but i cant get the 23.56 accuracty rate , i only get 33.01 accuracy rate!
> can you help me ?

I got the same problem.
